controllers/tasks.js

```
@@ -375,7 +375,7 @@ const updateTaskStatus = async (req, res, next) => {
         const isCurrProgress100 = parseInt(task.taskData.percentCompleted || 0) === 100;
         const isCurrProgress0 = parseInt(task.taskData.percentCompleted || 0) === 0;
         const isNewProgress100 = !!req.body.percentCompleted && parseInt(req.body.percentCompleted) === 100;
-        const isNewProgress0 = !!req.body.percentCompleted !== undefined && parseInt(req.body.percentCompleted) === 0;
+        const isNewProgress0 = req.body.percentCompleted !== undefined && parseInt(req.body.percentCompleted) === 0;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)